blockchain/blockchain.go

```diff
@@ -109,15 +109,13 @@ func (b *Blockchain) Store(block *core.Block, stateUpdate *core.StateUpdate) err
 	})
 }
 
-func (b *Blockchain) VerifyBlock(block *core.Block, stateUpdate *core.StateUpdate) error {
+func (b *Blockchain) IsNextHead(block *core.Block) error {
 	txn := b.database.NewTransaction(false)
 	defer txn.Discard()
-	return b.verifyBlock(txn, block, stateUpdate)
+	return b.isNextHead(txn, block)
```

Comment:
We should use `database.View ()` similar to other functions which were recently updated.